client/src/components/DirectoryAddUpdate.jsx

```diff
@@ -87,6 +87,52 @@ export default function DirectoryAddUpdate() {
     setErrors();
   }
 
+  // Check required fields to see if they're still blank 
+  const requiredFields = [coopName, websites, contactName, contactEmail, contactPhone, entityTypes];
+
+  const updateRequired = (field) => {
+    const asArray = Object.entries(errors);
+
+    let filteredItem = "";
+
+    switch(field) {
+      case coopName:
+        filteredItem = "coop_name";
+        break;
+      case websites:
+        filteredItem = "websites";
+        break;
+      case contactName:
+        filteredItem = "contact_name";
+        break;
+      case contactEmail:
+        filteredItem = "contact";
+        break;
+      case contactPhone:
+        filteredItem = "contact";
+        break;
+      case entityTypes:
+        filteredItem = "entity_types";
+        break;
+    }
+
+    if (errors.hasOwnProperty(filteredItem)) {
+      setErrors(Object.fromEntries(asArray.filter(([key, value]) => key !== filteredItem)));
+    };
+  }
+
+  const checkRequired = () => {
+    if (errors == undefined) {
```

Comment:
Any reason not to use "if (!errors) {" here instead of checking for only undefined?  If we only want to check for undefined, let's use "===" for consistency with other parts of the app.

Reply:
@laredotornado Thanks for the recommendation! I don't think there was a good reason, so that has been updated and it both errors out and submits correctly.